recipes/torchcodec/recipe.yaml

```
@@ -0,0 +1,132 @@
+
+build:
+  number: ${{ build_number }}
+  skip: 
+    # temporarily, we are running out of space in staged-recipes and will fix this in the feedstock
+    - cuda_compiler_version != "None"
+    # PyTorch does not support cuda 11
+    - cuda_compiler_version == "11.8"
+    # Upstream does not support Windows at the moment https://github.com/pytorch/torchcodec/issues/522
+    - win
+  string: ${{ torch_proc_type }}_py${{ python | version_to_buildstring }}_h${{ hash }}_${{ build_number }}
+  script:
+    file: build
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - pytorch
+        - if: cuda_compiler_version != "None"
+          then:
+            - cuda-driver-dev
+            - cuda-cudart-dev
+            - cuda-nvrtc-dev
+            - cuda-nvtx-dev
+            - cuda-nvml-dev
+            - cuda-profiler-api
+            - libcublas-dev
+            - libcufft-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib("c") }}
+    - if: cuda_compiler_version != "None"
+      then:
+        - ${{ compiler('cuda') }}
+    - cmake
+    - pkg-config
+    - make
+
+  host:
+    - python
+    - pip
+    - setuptools
+    - ffmpeg
+    # During the build, we link against the lgpl version of ffmpeg, to avoid license violation (and so we pass 
+    # I_CONFIRM_THIS_IS_NOT_A_LICENSE_VIOLATION to the wheel build to confirm that we want indeed to link to conda-forge's
+    # ffmpeg. At runtime the user can install either lgpl or gpl ffmpeg as preferred
+    - ffmpeg * [build=lgpl_*]
+    # This explicit pinning is a workaround for the fact that torchcodec 0.2.1 requires torch 2.6, but the pinned version is still 2.5.1
+    - libtorch 2.6.*
+    - libtorch * [build=${{ torch_proc_type }}*]
+    - pytorch 2.6.*
+    - pytorch * [build=${{ torch_proc_type }}*]
+
+    - if: cuda_compiler_version != "None"
+      then:
+        # These are transitive dependency of the libtorch cuda package
+        - cuda-driver-dev
+        - cuda-cudart-dev
+        - cuda-nvrtc-dev
+        - cuda-nvtx-dev
+        - cuda-nvml-dev
+        - cuda-profiler-api
+        - libcublas-dev
+        - libcufft-dev
+        - libcurand-dev
+        - libcusolver-dev
+        - libcusparse-dev
+        - magma
+        - nccl
+        - nvtx-c
+        # These are actual torchcodec deps
+        - libnpp-dev
+  run:
+    - python
+    - pytorch * [build=${{ torch_proc_type }}*]
+
+
+tests:
+  - python:
+      imports:
+        - torchcodec
+      pip_check: true
+
+  # test runs fine, but they increase the package size a lot as the
+  # test include big videos. So for now they are skipped, and it will
+  # be re-enabled in the feedstock with an appropriate strategy (such as
+  # packaging them as a separate `torchcodec-testdata`), for the time being the
```

I'd prefer to put the tests into `torchcodec-tests`; that'd match what we have for torchvision, but also other places like scipy etc.

---

Perfect, thanks. I will modiy the comment to make sure that we use that name.
```suggestion
  # packaging them as a separate `torchcodec-tests`), for the time being the
```